clients/cloud/scala/src/main/scala/io/confluent/examples/clients/scala/Consumer.scala

```diff
@@ -0,0 +1,32 @@
+package io.confluent.examples.clients.scala
+
+import java.time.Duration
+import java.util.{Collections, Properties}
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import scala.collection.JavaConversions._
+
+  object Consumer extends App {
+    var conf = ConfigFactory.load("ccloud.properties")
+    val props = propsFromConfig(conf)
+    val consumer = new KafkaConsumer[String, String](props)
+    consumer.subscribe(Collections.singletonList("testtopic"))
+
+    while(true) {
+      val records = consumer.poll(Duration.ofSeconds(1))
+      for (record <- records) {
+        System.out.println("Received message: (" + record.key() + ", " + record.value() + ") at offset " + record.offset())
```

Comment:
This should do the same per-user(key) summation that the other examples do, including json-deserialization

Reply:
done